src/Service/CloudDashboardServiceInterface.php

```diff
@@ -3,14 +3,14 @@
 namespace Drupal\cloud_dashboard\Service;
 
 /**
- * Class CloudDashboardService.
+ * Interface of CloudDashboardService.
```

Comment:
@RyoYamashita765 This can be:

```suggestion
 * An interface of CloudDashboardService.
```